### DISCUSSION TITLE
Access IIB when running independently using Python from another device

### DISCUSSION THREAD

**Discussion by 925-Studio** [zanllp/infinite-image-browsing#594] [Q&A] [answered]
I'm running this extension on Linux and I'd like to access it from another device. Is there any arguments I can use? Like ComfyUI has `--listen`. 
I mostly using ComfyUI but like checking my outputs in IIB, so it doesn't make much sense to use this extension through WebUI.
I've tried install independency pack on Linux but somehow the program doesn't open. So I like to stick with the extension.
Thanks in advance.

**Comment by zanllp** [comment, marked as answer]
`python app.py --port=8888 --host="0.0.0.0"`
<img width="1229" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/394e79bb-377f-4943-bfe9-fa5875e11c5b">

**Reply by 925-Studio** [reply]
Thank you! It works! And I will read the instruction more carefully in the future. 😁
